indico/modules/events/registration/templates/display/regform_display.html

```diff
@@ -85,7 +85,6 @@
             <div nd-reg-form
                 conf-id="{{ event.id }}"
                 conf-sections="{{ sections | tojson | forceescape }}"
-                conf-currency="{{ currency }}"
```

Comment:
I think it's cleaner getting it from item directly instead of passing it. I'll check how to do it.